sympy/physics/mechanics/kane.py

```diff
@@ -473,7 +473,8 @@ def to_linearizer(self):
         return Linearizer(f_0, f_1, f_2, f_3, f_4, f_c, f_v, f_a, q, u, q_i,
                 q_d, u_i, u_d, r)
 
-    def linearize(self, **kwargs):
+    # TODO : Remove `new_method` after 1.1 has been released.
+    def linearize(self, *, new_method=None, **kwargs):
```

Comment:
Is `new_method` used anywhere? 1.1 was released some time ago

Reply:
I've no idea - I figured I'd just leave the comment as it was. realistically, passing it should probably emit a deprecation warning.